src/distro/distro.py

```diff
@@ -1265,7 +1328,6 @@ def _distro_release_info(self) -> Dict[str, str]:
                     basename
                     for basename in os.listdir(self.etc_dir)
```

Comment:
I believe this access to `self.etc_dir` needs wrapping with `self.__resolve_chroot_symlink_as_needed`.